ext/spl/spl_heap.c

```diff
@@ -266,7 +291,7 @@ static void spl_ptr_heap_insert(spl_ptr_heap *heap, void *elem, void *cmp_userda
 	}
 
 	/* sifting up */
```

Comment:
```suggestion
	/* shifting up */
```

Reply:
"sift" is correct here.

Reply:
TIL: oh wow... had to look this one up in the dictionary.